src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carRentOfDay = 40;
+  const longTermRent = 7;
+  const shortTermRent = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
```

Comment:
The constants are well-named, making the code self-explanatory. This is a good example of avoiding hard-coded values in the code.